flink-state-backends/flink-statebackend-changelog/src/main/java/org/apache/flink/state/changelog/ChangelogKeyedStateBackend.java

```diff
@@ -116,16 +140,54 @@
     /** For caching the last accessed partitioned state. */
     private String lastName;
 
+    private final FunctionDelegationHelper functionDelegationHelper =
+            new FunctionDelegationHelper();
+
+    /** Updated initially on restore and later upon materialization (after FLINK-21356). */
+    private final List<KeyedStateHandle> materialized = new ArrayList<>();
+
+    /** Updated initially on restore and later cleared upon materialization (after FLINK-21356). */
```

Comment:
What does `after FLINK-21356` means? If I understand correctly, this `restoredNonMaterialized` would be cleared upon materialization which might be implemented in FLINK-21357, right? If so, why not change the doc to `in FLINK-21357`?
The same comment for `materialized` field.

Reply:
Right, I'll change the comment accordingly.